src/apd/util_test.go

```diff
@@ -0,0 +1,43 @@
+package apd
+
+import (
+	"testing"
+
+	"github.com/SkycoinProject/skycoin/src/cipher"
+)
+
+func TestBroadCastPubKey(t *testing.T) {
+	type args struct {
+		PublicKey   string
+		BroadCastIP string
+		Port        int
+	}
+	publicKey := func() string {
+		key, _ := cipher.GenerateKeyPair()
+		return key.Hex()
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			"success",
+			args{
+				publicKey(),
+				"255.255.255.255",
+				3000,
+			},
+			false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
If we add more cases to this test, `tt` will contain a reference to the loop variable and get incorrect value in subtests. With one test case, it should work, but I would fix it to avoid bugs in the future.

```suggestion
		tt := tt
		t.Run(tt.name, func(t *testing.T) {
```